lib/encoder/include/Pololu37D.h

```diff
@@ -5,11 +5,12 @@
 
 namespace Encoder {
 class Pololu37D final : public Encoder {
+  // link to the encoder:
```

Comment:
@cindyli-13  i remember u wanted to rename this encoder class. Any suggestions?

Reply:
How bout `Quadrature64CPR`

Reply:
sure. please leave a comment saying this motor is connected to the pololu 37d motors

Reply:
just to confirm would you like me to rename the class, including the header and src file?

Reply:
yup